src/main/java/com/googleinterns/zoomtube/data/Comment.java

```diff
@@ -75,6 +81,8 @@ public static Builder builder() {
     public abstract Builder setParentKey(Optional<Key> parentKey);
     public abstract Builder setTimestampMs(long timestampMs);
     public abstract Builder setTimestampMs(Optional<Long> timestampMs);
+    public abstract Builder setTranscriptLineKey(Key transcriptLineKey);
+    public abstract Builder setTranscriptLineKey(Optional<Key> transcriptLineKey);
```

Comment:
Why are these methods duplicated? You only need 1.

Reply:
I am having some compiler errors when I only include one of them...

Reply:
Removed the non-optional one after talking offline